src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -121,6 +121,7 @@
     .pill-selector-input-wrapper {
         border: 1px solid $seesee;
         margin-top: 10px;
+        min-height: 75px;
```

Comment:
This should be the same hight for the input fields used in the forms for send invite collaborators and email shared link forms in the USM. ideally, they'd all get this new `min-height` from a shared variable, instead of it being specified several times.

Reply:
Will update with a shared variable.